quantum/rgb_matrix.c

```diff
@@ -23,6 +23,7 @@
 #include "eeprom.h"
 #include <string.h>
 #include <math.h>
+#include "rgblight_list.h"
```

Comment:
Actually, this should go in `rgb_matrix.h`, that way, it's picked up everywhere.

Reply:
Thanks for the info.
Moved it to the header file.